examples/volumes/flexvolume/dummy-attachable

```diff
@@ -0,0 +1,104 @@
+#!/bin/sh
+
+# Copyright 2017 The Kubernetes Authors.
```

Comment:
examples are moved to https://github.com/kubernetes/examples and `examples/` directory soon will be removed. consider submitting this to the examples repository instead.